doc/host_config_schema/cchost_config.json

```diff
@@ -58,6 +58,12 @@
                 "type": "integer",
                 "default": 1010,
                 "description": "The maximum number of active client sessions on that interface after which clients sessions will be terminated, before the TLS handshake is complete. Note that its value must be greater than the value of ``max_open_sessions_soft``"
+              },
+              "endorsement_type": {
```

Comment:
An alternative would be `"endorsement"` with a single member `"type"`. I didn't do that for simplicitly, but that may be short-sighted.

Reply:
Yes!

Reply:
Offline consensus: `"endorsement"` as object with single field (for now) `"authority"` being an enum of values `"Node"`, `"Service"`.